content/microservices/redis.md

```diff
@@ -26,11 +26,47 @@ const app = await NestFactory.createMicroservice(ApplicationModule, {
 });
 ```
 
-> info **Hint** `Transport` enumerator is imported from the `@nestjs/microservices` package.
+> info **Hint** `Transport` enum is imported from the `@nestjs/microservices` package.
+
+Likewise, to create a client instance, we need to pass the options object which has the same properties we saw above in the `createMicroservice()` method.
```

Comment:
Likewise, to create a client instance, we need pass an options object with the same properties we saw above in the `createMicroservice()` method.